config-generator/src/main/resources/templates/infinispan.xml

```diff
@@ -57,7 +60,9 @@
     <cache-container name="default" statistics="true" zero-capacity-node="{infinispan.zeroCapacityNode}">
         <transport cluster="$\{infinispan.cluster.name:{infinispan.clusterName}\}" node-name="$\{infinispan.node.name:\}"
         {#if xsite.backups}stack="xsite"{#else}stack="image-{jgroups.transport}"{/if}/>
-
+        {#if cloudEvents}
+            <ce:cloudevents {#if cloudEvents.boostrapServer}bootstrap-servers="{cloudEvents.bootstrapServer}"{/if} {#if cloudEvents.acks}acks="{cloudEvents.acks}"{/if} {#if cloudEvents.cacheEntriesTopic}cache-entries-topic="{cloudEvents.cacheEntriesTopic}"{/if}/>
```

Comment:
Isn't `bootstrap-servers` mandatory? In which case we can just do `bootstrap-servers="{cloudEvents.bootstrapServer}"`


This also applies to acks and cache-entries-topic if they are mandatory, but I'm not sure if that's the case?

Reply:
uh yeah `bootstrap-servers` is mandatory, `acks` and `cache-entries-topic` have defaults value if not provided.